script/utils/bookUtils.ts

```diff
@@ -35,7 +35,9 @@ export async function findBooks({
 
   const bookInfo = bookId
     ? [{id: bookId, version: bookVersion || assertDefined(bookConfig[bookId], '').defaultVersion}]
-    : Object.entries(bookConfig).map(([id, {defaultVersion}]) => ({id, version: defaultVersion}))
+    : Object.entries(bookConfig)
+      .filter(([, book]) => !book.retired)
+      .map(([id, {defaultVersion}]) => ({id, version: defaultVersion}))
```

Comment:
@TomWoodward this util is used by the `domVisitor` and `urlChecker` scripts

Reply:
wait... do we want urlChecker to run on retired books?